tests/CppUTestExt/MockFailureTest.cpp

```diff
@@ -205,9 +205,9 @@ TEST(MockFailureTest, MockExpectedParameterDidntHappenFailure)
     MockExpectedParameterDidntHappenFailure failure(UtestShell::getCurrent(), "foo", *list);
     STRCMP_EQUAL("Mock Failure: Expected parameter for function \"foo\" did not happen.\n"
                  "\tEXPECTED calls that DID NOT happen related to function: foo\n"
-                 "\t\tfoo -> int bar: <2>, const char* boo: <str>\n"
+                 "\t\tfoo -> int bar: <2 (0x00000002)>, const char* boo: <str>\n"
                  "\tACTUAL calls that DID happen related to function: foo\n"
-                 "\t\tfoo -> int bar: <10>, const char* boo: <bleh>\n"
+                 "\t\tfoo -> int bar: <10 (0x0000000a)>, const char* boo: <bleh>\n"
```

Comment:
If this is really how CppUTest displays the value, then we have another inconsistency, because elsewhere small values would be displayed like, e.g. (0x1) or (0xa)